nsxt/resource_nsxt_policy_group.go

```diff
@@ -844,12 +855,18 @@ func resourceNsxtPolicyGroupCreate(d *schema.ResourceData, m interface{}) error
 	description := d.Get("description").(string)
 	tags := getPolicyTagsFromSchema(d)
 
+	var groupTypes []string
+	groupType := d.Get("group_type").(string)
+	if groupType != "" {
+		groupTypes = append(groupTypes, groupType)
+	}
 	obj := model.Group{
 		DisplayName:        &displayName,
 		Description:        &description,
 		Tags:               tags,
 		Expression:         expressionData,
 		ExtendedExpression: extendedExpressionList,
+		GroupType:          groupTypes,
```

Comment:
I believe version check is needed here since GroupType support was added recently in NSX